recipes/geoalchemy2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "geoalchemy2" %}
+{% set camelName = "GeoAlchemy2"}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ camelName }}-{{ version }}.tar.gz
```

Comment:
Ouch! The URL needs the Camel Case Name!! Sorry...

PS: I am not a big fan use too much `jinja2`. Sometimes it adds more noise than it is worth. (That is taste and you do not need to change that on my account.)